docs/devupdate.md

```diff
@@ -9,3 +9,9 @@ tags:
 import { CodeBlock, CodeExample, InvalidExample, ValidExample } from '@site/src/components';
 
 This page highlights the important changes that are coming in Moodle 4.3 for developers.
+
+## Functions for generating and displaying user details
+
+#### `core_user::get_profile_picture` for retrieving user picture.
```

Comment:
Please use lists for the methods instead of headings. 

Also, something to remember for the future: For accessibility reasons, headings must increase by one level. e.g. an `h2` heading needs to have an `h3` heading underneath it.